src/test/javascript/spec/component/guided-tour/guided-tour.component.spec.ts

```diff
@@ -120,6 +120,15 @@ describe('Component Tests', () => {
                 expect(guidedTourComponent.currentTourStep).to.exist;
             });
 
+            it('should navigate next with the right arrow key', () => {
```

Comment:
The name of the test is a copy&paste error.

Reply:
You're right. I changed that :)